docs/DeveloperGuide.md

```diff
@@ -245,6 +245,65 @@ Due to the repercussions of Alternative 2 and the efficiency of Alternative 1, w
 
 ### 3.3 Show feature
 
+The show feature enables users to show tasks by specifying the given week or date.
+
+#### 3.3.1 Implementation
+
+The following are the changes made to achieve this feature:
+
+* A `` class is added under the `model/task` package.
```

Comment:
Did u miss out 'show' in between the '`' '`'

Reply:
Have explained in the group